packages/cms-lib/api/fileMapper.js

```diff
@@ -91,7 +91,7 @@ async function fetchFileStream(portalId, filePath, destination, options = {}) {
   const response = await http.getOctetStream(
     portalId,
     {
-      uri: `${FILE_MAPPER_API_PATH}/stream/${filePath}`,
+      uri: `${FILE_MAPPER_API_PATH}/stream${filePath}`,
```

Comment:
Is the assumption that `filePath` will always be prefixed with `/`? Are we sure that is true?

Reply:
@miketalley You could extract this logic into a reusable function to format and ensure slashes.
https://github.com/HubSpot/hubspot-cms-tools/blob/3c1cd1c4391189af2dbb61187632858789047ce8/packages/cms-lib/api/fileMapper.js#L16-L21

Reply:
I tried it with several files/directories/subdirectories and it works fine. This was just a superflous `/` that I removed, but I suppose it doesn't hurt to leave it either. Thoughts?

Reply:
The logic I linked above is already in this file. I'd suggest pulling extracting it to as helper to be called upon `filePath`.